ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-non-prod.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "AccessControl": [
+      {
+        "UserRoles": [
+
+          "caseworker-civil-judge"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "caseworker-civil-admin"
+        ],
+        "CRUD": "CRU"
+      },
```

Comment:
Should "caseworker-civil-judge" & "caseworker-civil-admin" not be combined into the same userRole section considering they have the same CRUD permissions? 

Something like:
"AccessControl": [
      {
        "UserRoles": [

          "caseworker-civil-judge",
           "caseworker-civil-admin"
        ],
        "CRUD": "CRU"
      },

Reply:
Updated the UserRoles